app/payments/RazorpayPayment.php

```diff
@@ -0,0 +1,164 @@
+<?php
+use Razorpay\Api\Api;
+
+class RazorpayPayment
+{
+    public function getSessionValue($key)
+    {
+        return $_SESSION[$key];
+    }
+
+    public function setSessionValues($sessionValues)
+    {
+        foreach($sessionValues as $key => $value)
+        {
+            $_SESSION[$key] = $value;
```

Comment:
opencart recommends using $this->session instead: https://stackoverflow.com/questions/8472027/session-variables-in-opencart